site-src/guides/implementers.md

```diff
@@ -53,7 +53,7 @@ spec:
   extensionRef:
     name: vllm-llama3-8b-instruct-epp
 ```
-
+mkdocs.yml
```

Comment:
Guessing this was unintentional? Since we already split conformance tests out into a separate guide, we could do the same here and make a corresponding change to mkdocs.yml. Don't think this is a blocker, will defer to you here.